PhysicsTools/TensorFlow/test/testBase.h

```diff
@@ -8,6 +8,7 @@
 #define PHYSICSTOOLS_TENSORFLOW_TEST_TESTBASE_H
 
 #include <boost/filesystem.hpp>
```

Comment:
This can be also removed, as well as all occurrencies in ​PhysicsTools/​TensorFlow/​test/​BuildFile.xml

Reply:
this I kept as I was not able to find a simple replacement for `boost::filesystem::unique_path()` which generates a random `xxxx-xxxx-xxxx-xxxx` string